src/components/metrics/card/SensitivitySection/SensitivitySection.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component, PropTypes} from 'react'
+import {BarChart, ScatterPlot} from 'react-d3-components'
+import {sampleMean, sampleStdev, percentile, cutoff, sortDescending} from 'lib/dataAnalysis.js'
+import everpolate from 'everpolate'
+import './style.css'
+
+function importance(r2) {
+  if (r2 < 0.05) { return 'low' }
+  else if (r2 < 0.5) { return 'medium' }
+  else { return 'high' }
+}
+
+class Plot extends Component {
+  render() {
+    const {xSamples, ySamples, size, xLabel, yLabel} = this.props
+    const data = [{ customValues: _.zip(xSamples, ySamples) }];
+    const valuesAccessor = (s) => s.customValues
+    const xAccessor = (e) => e[0]
+    const yAccessor = (e) => e[1]
+
+    if (size === 'SMALL'){ return(
+          <ScatterPlot
+            data={data}
+            width={180}
+            height={68}
+            margin={{top: 5, bottom: 9, left: 5, right: 3}}
+            xAxis={{tickArguments: [0], innerTickSize: 1, outerTickSize: 1, tickPadding: 1}}
+            yAxis={{tickArguments: [0], innerTickSize: 0, outerTickSize: 0, tickPadding: 0}}
+            x={xAccessor}
+            y={yAccessor}
+            values={valuesAccessor}/>
+    )}
+    else { return(
+      <ScatterPlot
+        data={data}
+        width={500}
+        height={300}
+        margin={{top: 10, bottom: 40, left: 60, right: 20}}
+        xAxis={{tickArguments: [6], innerTickSize: 5, outerTickSize: 2, tickPadding: 3, label: xLabel}}
+        yAxis={{tickArguments: [6], innerTickSize: 5, outerTickSize: 2, tickPadding: 3, label: yLabel}}
+        x={xAccessor}
+        y={yAccessor}
+        values={valuesAccessor}/>
+    )}
+  }
+}
+
+class RegressionStats extends Component {
+  render() {
+    const {xSamples, ySamples, size} = this.props
```

Comment:
It is strange to me how you have the defensive coding checks for xSamples and not ySamples. What happens if ySamples are undefined? 

Also, can you just pull all your checks to the top? e.g.
if (!xSamples && !xSamples.length && !ySamples && !ySamples.length) {
  return <div></div>
}
...
?